openlibrary/plugins/openlibrary/js/utils.js

```diff
@@ -1,7 +1,7 @@
 // http://jqueryminute.com/set-focus-to-the-next-input-field-with-jquery/
 $.fn.focusNextInputField = function() {
     return this.each(function() {
-        var fields = $(this).parents('form:eq(0),body').find(':input:visible');
+        var fields = $(this).parents('form').eq(0).add('body').find(':input:visible');
```

Comment:
Today I've learned something :)
Thank you!

Reply:
This makes me wonder if we should be doing this, but a task for another time :)

> Because :input is a jQuery extension and not part of the CSS specification, queries using :input cannot take advantage of the performance boost provided by the native DOM querySelectorAll() method. To achieve the best performance when using :input to select elements, first select the elements using a pure CSS selector, then use .filter(":input").